### ISSUE TITLE
iOS: Avatar and notification icons 

### ISSUE THREAD

**Issue by samuelclay** [samuelclay/NewsBlur#1740]
See this screenshot. Happens on both iPhone 14 and iPhone SE. Avatar photo is too low and the notification icon is cut off.

![Simulator Screen Shot - iPhone 14 Pro - 2022-10-20 at 10 03 33](https://user-images.githubusercontent.com/44229/196970726-735d6279-4136-478c-bd6e-22095af8b1f9.png)

**Comment by Dejal** [comment]
I've tweaked the position of the avatar icon, but I can't do anything about the notification icon; that is just how the SVG is rendered by Xcode.

Here's Xcode's Images.xcassets:

<img width="841" alt="Xcode" src="https://user-images.githubusercontent.com/1318849/197017255-bd7b221e-3608-4f6e-b939-834091a5f4a5.png">

**Comment by samuelclay** [comment]
Ok, last thing to fix. I updated the notifications icon but it's positioned too far to the right and the avatar is also too far to the right. The avatar should be left aligned with the icons below, and the notifications icon should be aligned with the popover.

![IMG_4A6BDA18B655-1](https://user-images.githubusercontent.com/44229/198048425-fd704903-0eaa-40bb-ae62-bcc3e4865b91.jpeg)

**Comment by samuelclay** [comment]
For the notification icon, can we make the button physically wider and left-align the notifications image in it?

**Comment by Dejal** [comment]
I've moved the avatar as far to the left as I can, and moved the notifications to line up with the popover (as it was before I moved it to the right).

![Buttons](https://user-images.githubusercontent.com/1318849/198118006-33970664-667b-430d-adb4-733c901da295.png)

Here's a screenshot that might help explain the situation: the blue box shows the avatar view (the navigation bar title view) and the red box shows the notifications button (the right bar button):

![Frames](https://user-images.githubusercontent.com/1318849/198117978-507b3f03-8540-4039-bb18-02290912fbab.png)

The avatar button can't be further to the left; I can draw the icon outside that box, but then tapping part of it won't do anything, which isn't a great experience.

Similarly, I can draw the notification icon further to the right, but can't move or resize the button, leading to the popover not lining up, as you saw. So it really needs to be centered in the box, as now.

I've left the background color code commented out in the code, if you want to try it yourself.
